rel/overlay/etc/default.ini

```diff
@@ -188,6 +188,10 @@ credentials = false
 ; List of accepted methods
 ; methods =
 
+[prefer_header]
```

Comment:
Bike shed: Is this the best place for this, or would it be better as a key in the chttpd section. Something like "prefer_minimal = List-Of-Headers"

You might also add a comment that a user doesn't specify Server here that Mochiweb will add its own regardless.